site/docs/spec.md

```diff
@@ -967,6 +968,7 @@ Table metadata is serialized as a JSON object according to the following table.
 |Metadata field|JSON representation|Example|
 |--- |--- |--- |
 |**`format-version`**|`JSON int`|`1`|
+|**`object-type`**|`JSON string`|`table`|
```

Comment:
Nit: There should be double quotes around `table` because the example should be a JSON string, like the other strings below.